library/src/main/res/values/library_context_menu_android_strings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_int_context_menu.android"></string>
```

Comment:
still everywhere `define_int_context_menu.android` and not `define_int_context_menu_android`